src/lib/forward.ts

```diff
@@ -35,6 +35,17 @@ export class EventForwarder {
       this.setupNode(node),
     );
 
+    // Bind to driver events
+    this.clientsController.driver.on("driver ready", () => {
+      // forward event to all connected clients, respecting schemaVersion it supports
```

Comment:
a) this comment is not accurate with the code right now and
b) we should bump the schema and only send this when its >=40 so that we don't break older clients